src/Core/Silk.NET.SilkTouch/NativeApiGenerator.cs

```diff
@@ -346,29 +361,289 @@ void BuildLoadInvoke(ref IMarshalContext ctx, Action next)
+                    (
+                        List<AttributeListSyntax>(), TokenList(Token(SyntaxKind.PrivateKeyword)),
+                        VariableDeclaration
+                        (
+                            IdentifierName("Silk.NET.Core.Contexts.INativeContext"),
+                            SingletonSeparatedList(VariableDeclarator("_ctx"))
+                        )
+                    )
+                );
+
+                vTableMembers.Add
+                (
+                    MethodDeclaration(PredefinedType(Token(SyntaxKind.VoidKeyword)), "Initialize")
+                        .WithModifiers(TokenList(Token(SyntaxKind.PublicKeyword)))
+                        .WithParameterList
+                        (
+                            ParameterList
+                            (
+                                SeparatedList
+                                (
+                                    new[]
+                                    {
+                                        Parameter
+                                                (Identifier("ctx"))
+                                            .WithType(IdentifierName("Silk.NET.Core.Contexts.INativeContext")),
+                                        Parameter
+                                                (Identifier("maxSlots"))
+                                            .WithType(PredefinedType(Token(SyntaxKind.IntKeyword)))
+                                    }
+                                )
+                            )
+                        )
+                        .WithBody
+                        (
+                            Block
+                            (
+                                ExpressionStatement
+                                (
+                                    AssignmentExpression
+                                    (
+                                        SyntaxKind.SimpleAssignmentExpression, IdentifierName("_ctx"),
+                                        IdentifierName("ctx")
+                                    )
+                                )
+                            )
+                        )
+                );
+
+                List<VariableDeclaratorSyntax> slotVars = new List<VariableDeclaratorSyntax>();
+                List<StatementSyntax> loadStatements = new List<StatementSyntax>();
+                foreach (var entrypoint in entryPoints)
+                {
+                    var name = $"_{entrypoint}";
+                    slotVars.Add(VariableDeclarator(name));
+                    loadStatements.Add
+                    (
+                        IfStatement
+                        (
+                            BinaryExpression
+                            (
+                                SyntaxKind.EqualsExpression, IdentifierName("entryPoint"),
+                                LiteralExpression(SyntaxKind.NumericLiteralExpression, Literal(entrypoint))
```

Comment:
As discussed in Discord, in the next PR this should do slot comparisons instead of entryPoint comparisons.